vocabularies/qld-resource-permit-status.ttl

```diff
@@ -0,0 +1,272 @@
+    skos:broader gmps:granted ;
+    skos:inScheme gmps:conceptScheme ;
+    skos:prefLabel "Accepted"@en .
+
+gmps:application-lapsed a skos:Concept ;
+    skos:definition "The application for the Resource Permit has lapsed."@en ;
+    skos:broader gmps:non-current ;
+    skos:inScheme gmps:conceptScheme ;
+    skos:prefLabel "Application lapsed"@en .
+
+gmps:awaiting-payment-confirmation a skos:Concept ;
+    skos:definition "Payment has been made, but the Department needs confirmation of this."@en ;
+    skos:broader gmps:submitted ;
+    skos:inScheme gmps:conceptScheme ;
+    skos:prefLabel "Awaiting payment confirmation"@en .
+
+gmps:cancelled a skos:Concept ;   
+    skos:definition "The Resource Permit has been cancelled."@en ;
+    skos:broader gmps:non-current ;
+    skos:inScheme gmps:conceptScheme ;
+    skos:prefLabel "Cancelled"@en .
+
+gmps:competing-application a skos:Concept ;
+    skos:definition "This application represents a competing application."@en ;
+    skos:broader gmps:application,
+        gmps:granted ;
+    skos:inScheme gmps:conceptScheme ;
+    skos:prefLabel "Competing application"@en .
+
+gmps:conditionally-surrendered a skos:Concept ;
+    skos:definition "The application has been conditionally surrendered."@en ;
+    skos:broader gmps:application,
+        gmps:non-current ;
+    skos:inScheme gmps:conceptScheme ;
+    skos:prefLabel "Conditionally surrendered"@en .
+
+gmps:conversion-of-ml-to-mc a skos:Concept ;
+    skos:definition "The Mining Lease has been converted to a Mining Claim."@en ;
+    skos:broader gmps:granted ;
+    skos:inScheme gmps:conceptScheme ;
+    skos:prefLabel "Conversion of ML to MC"@en .
+
+gmps:deemed-abandoned a skos:Concept ;
+    skos:definition "The Resource Permit is deemed to be abandoned."@en ;
+    skos:broader gmps:non-current ;
+    skos:inScheme gmps:conceptScheme ;
+    skos:prefLabel "Deemed abandoned"@en .
+
+gmps:expired a skos:Concept ;
+    skos:definition "The Resource Permit has expired."@en ;
+    skos:broader gmps:granted,
+        gmps:non-current ;
+    skos:inScheme gmps:conceptScheme ;
+    skos:prefLabel "Expired"@en .
+
+gmps:finalised a skos:Concept ;
+    skos:definition "The Resource Permit has no outstanding requirements."@en ;
+    skos:broader gmps:non-current ;
+    skos:inScheme gmps:conceptScheme ;
+    skos:prefLabel "Finalised"@en .
+
+gmps:in-progress a skos:Concept ;
+    skos:definition "The Application is in-progress."@en ;
+    skos:broader gmps:discarded,
+        gmps:prelodgement ;
+    skos:inScheme gmps:conceptScheme ;
+    skos:prefLabel "In progress"@en .
+
+gmps:on-hold-oil-shale-application a skos:Concept ;
+    skos:definition "The oil-shale application has been placed on-hold."@en ;
+    skos:broader gmps:application ;
+    skos:inScheme gmps:conceptScheme ;
+    skos:prefLabel "On-hold oil shale application"@en .
+
+gmps:paid a skos:Concept ;
+    skos:definition "Application payment has been made."@en ;
```

Comment:
In contrast to awaiting-payment-confirmation should this be "Application payment has been made and confirmation of payment received"?